.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: ['plugin:react/recommended', 'standard', 'prettier-standard'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 12,
+    sourceType: 'module'
+  },
+  plugins: ['react', '@typescript-eslint'],
+  settings: {
+    react: {
+      version: 'detect'
+    }
+  },
+  rules: {
+    'no-use-before-define': 'off'
+  }
```

Comment:
with React 17 we don't need need to import `React` https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#nextjs so we should add these 2 rules
```js
    "react/jsx-uses-react": "off",
    "react/react-in-jsx-scope": "off",
```

Reply:
Awesome, didn't know that. I'll update it.